### PR TITLE
[link prober] Increase pause/restart probe log verbosity

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] New feature
- [ ] Doc/Design
- [ ] Unit test

#### What is the motivation for this PR?
As the subject.
Work item tracking
Microsoft ADO (number only): 24609952

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>

#### How did you do it?
Increase log level to warning so probe pause/restart could be observed in syslog.

### CODE REVIEW

File: src/link_prober/LinkProber.cpp [hunk 1]
```diff
@@ -865,14 +865,14 @@ void LinkProber::resetIcmpPacketCounts()
 
 void LinkProber::shutdownTxProbes()
 {
-    MUXLOGDEBUG(mMuxPortConfig.getPortName());
+    MUXLOGWARNING(boost::format("%s: shutdown probe") % mMuxPortConfig.getPortName());
```
Comment (zjswhhh) [inline src/link_prober/LinkProber.cpp:868]: Let's be more specific and say "shutdown ICMP heartbeat probing".

**Review by zjswhhh — COMMENTED** [review]
Also, shall we lift the log level for suspending heartbeat as well.

**Review by yxieca — APPROVED** [review]

**Review by zjswhhh — APPROVED** [review]
lgtm

**Pull request merged**